docs/includes/topic_f5lbaas-l7_content_switching.rst

```diff
@@ -4,9 +4,9 @@ F5 LBaaSv2 L7 Content Switching
 Overview
 --------
 
-L7 content switching takes its name from layer 7 of the OSI Model, also called the application layer. [#f1]_ As the name implies, L7 content switching decisions are based on the application data, or content, of request traffic as it passes through the virtual server. Via the API, you can define actions to be taken when certain content conditions are met. See the OpenStack Neutron documentation <http://specs.openstack.org/openstack/neutron-specs/specs/mitaka/lbaas-l7-rules.html>_ for more information.
+L7 content switching takes its name from layer 7 of the OSI Model, also called the application layer. [#]_ As the name implies, L7 content switching decisions are based on the application data, or content, of request traffic as it passes through the virtual server. Via the API, you can define actions to be taken when certain content conditions are met. See the OpenStack Neutron documentation <http://specs.openstack.org/openstack/neutron-specs/specs/mitaka/lbaas-l7-rules.html>_ for more information.
```

Comment:
"OpenStack Neutron `LBaaS Layer 7 rules documentation <http://specs.openstack.org/openstack/neutron-specs/specs/mitaka/lbaas-l7-rules.html>`_"
(the opening and closing ` ` are missing; I just added the name of the document for clarification)